catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/utils/DorisUtils.java

```diff
@@ -32,7 +32,7 @@ public static Map<String, String> extractPropertiesFromSql(String createTableSql
     String[] lines = createTableSql.split("\n");
 
     boolean isProperties = false;
-    final String sProperties = "\"(.*)\"\\s{0,}=\\s{0,}\"(.*)\",?";
+    final String sProperties = "\"(.*)\"\\s*=\\s*\"(.*)\",?";
```

Comment:
Does it looks more elegant to use `*` compared to `{0,}`, I can't tell the difference between them.

Reply:
According to Justin's suggestion, `*` is more concis.

Reply:
OK, let's just follow his suggestion.